src/acat/azext_acat/custom.py

```diff
@@ -0,0 +1,537 @@
+
+    def _execute_operations(self):
+        self.pre_operations()
+        yield self.OnboardAcatWithDupAadToken(ctx=self.ctx)()
+        self.post_operations()
+
+
+class TriggerEvaluation(_AcatTriggerEvaluation):
+
+    class TriggerEvaluationWithDupAadToken(
+        _AcatTriggerEvaluation.ProviderActionsTriggerEvaluation
+    ):
+        CLIENT_TYPE = "AcatMgmtClient"
+
+    def _execute_operations(self):
+        self.pre_operations()
+        yield self.TriggerEvaluationWithDupAadToken(ctx=self.ctx)()
+        self.post_operations()
+
+
+class ListAcatReport(_AcatListReport):
+    class ListAcatReportWithDupAadToken(_AcatListReport.ReportList):
+        CLIENT_TYPE = "AcatMgmtClient"
+
+    def _execute_operations(self):
+        self.pre_operations()
+        self.ListAcatReportWithDupAadToken(ctx=self.ctx)()
+        self.post_operations()
+
+
+class ShowAcatReport(_AcatShowReport):
+    class ShowAcatReportWithDupAadToken(_AcatShowReport.ReportGet):
+        CLIENT_TYPE = "AcatMgmtClient"
+
+    def _execute_operations(self):
+        self.pre_operations()
+        self.ShowAcatReportWithDupAadToken(ctx=self.ctx)()
+        self.post_operations()
+
+
+class CreateAcatReport(_AcatCreateReport):
+    class CreateAcatReportWithDupAadToken(_AcatCreateReport.ReportCreateOrUpdate):
+        CLIENT_TYPE = "AcatMgmtClient"
+
+    def _execute_operations(self):
+        self.pre_operations()
+        yield self.CreateAcatReportWithDupAadToken(ctx=self.ctx)()
+        self.post_operations()
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        args_schema = super()._build_arguments_schema(*args, **kwargs)
+        from azure.cli.core.aaz import AAZDateTimeArg, AAZStrArg
+
+        args_schema.time_zone._required = False
+        args_schema.time_zone._registered = False
+        args_schema.time_zone_with_default = AAZStrArg(
+            options=["--time-zone"],
+            arg_group="Properties",
+            help="""Report collection trigger time\'s time zone, the available list can be obtained by executing
+            "Get-TimeZone -ListAvailable" in PowerShell. An example of valid timezone id is "Pacific Standard Time".""",
+            default="UTC",
+        )
+        args_schema.trigger_time._required = False
+        args_schema.trigger_time._registered = False
+        args_schema.trigger_time_with_default = AAZDateTimeArg(
+            options=["--trigger-time"],
+            arg_group="Properties",
+            help="Report collection trigger time.",
+            default=GetClosestFullHour(),
+        )
+        return args_schema
+
+    def pre_operations(self):
+
+        poller = OnboardAcat(cli_ctx=self.cli_ctx)(
+            command_args={
+                "subscription_ids": ParseSubsFromResources(self.ctx.args.resources),
+            }
+        )
+        LongRunningOperation(self.cli_ctx)(poller)
+        args = self.ctx.args
+        args.time_zone = args.time_zone_with_default
```

Comment:
You can set the default value in aaz-dev-tools directly, then you can remove the time_zone_with_default arg